gpii/node_modules/transformer/test/html/TransformerTests.html

```diff
@@ -15,6 +15,12 @@
         <script src="../../../../../node_modules/infusion/src/framework/core/js/ModelTransformation.js"></script>
         <script src="../../../../../node_modules/infusion/src/framework/core/js/ModelTransformationTransforms.js"></script>
 
+        <!-- These are required not for the tests themselves, but for the coverage client -->
+        <script type="text/javascript" src="../../../../../node_modules/infusion/src/framework/core/js/FluidDocument.js"></script>
```

Comment:
This mix of the two styles is not unique to my work, but I see your point and am happy to be the one who cleans it up.